src/FffGcodeWriter.cpp

```diff
@@ -3986,14 +3937,12 @@ void FffGcodeWriter::setExtruder_addPrime(const SliceDataStorage& storage, Layer
 
 void FffGcodeWriter::addPrimeTower(const SliceDataStorage& storage, LayerPlan& gcode_layer, const size_t prev_extruder) const
 {
-    if (! storage.primeTower.enabled_)
+    if (storage.prime_tower_)
```

Comment:
Guess it's a personal preference (it's not one that is in our code standard as far as I'm aware) but i tend to prefer the early out option (so the return if the primetower is not set). The reason for that is that you need less indented code, which especially for more complicated code means you're going less deep.

 I'm okay with leaving it the way you have it as I don't feel that strong about it.

Reply:
You're totally right, the CuraEngine code makes use of early out most often, and I am personally not used to this, so I sometimes forget it. I'll change it for consistency.